docker-compose.windows.yml

```diff
@@ -28,7 +28,6 @@ services:
   smoke-tests.windows:
     build:
       context: ./tracer/ # have to use this as the context, as Dockercompose requires dockerfile to be inside context dir
-      target: publish-msi
```

Comment:
I was going to try and reuse the `smoke.windows.dockerfile` for this check, but that wasn't really possible in the end, so this target isn't necessary